src/test/java/com/meilisearch/integration/SearchTest.java

```diff
@@ -147,11 +147,37 @@ public void testSearchCrop() throws Exception {
         SearchRequest searchRequest =
                 new SearchRequest("and")
                         .setAttributesToCrop(new String[] {"overview"})
-                        .setCropLength(5);
+                        .setCropLength(1);
 
-        SearchResult searchResult = index.search(searchRequest);
+        Results res_gson = jsonGson.decode(index.rawSearch(searchRequest), Results.class);
 
-        assertEquals(20, searchResult.getHits().size());
+        assertEquals(20, res_gson.hits.length);
+        assertEquals(5, res_gson.hits[0].getFormatted().getOverview().length());
```

Comment:
`.length()` is a function or a field? Because in the line above you used as a field 🤔 (probably is related to the getOverview response, but just checking out)